packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,33 @@
+This tutorial will be exploring different ways of rendering a list of
+items in Lit!
+
+## Previous knowledge
+It's recommended to have an understanding of
+[defining a component](/docs/components/defining/),
+[rendering](/docs/components/rendering/), and
+[reactive properties](/docs/components/properties/) before
+working on this tutorial. The [Intro to Lit](/tutorials/intro-to-lit/) tutorial
+is a great place to start.
+
+
+## Rendering a list
+Begin by rendering a list of names from an array. The example already has set
```

Comment:
I think there's a bit of conceptual info that could be added here to make list support click for devs.

"List" support is really a combination of two things: lit-html supports nested template results as values, and it supports iterables of any supported value type. Combine those and you get iterables of template results.

It might be nice to start with something _very_ simple, like:

```ts
  render() {
    return html`
      <p>A list of my friends</p>
      ${['Chandler', 'Phoebe', 'Joey', 'Monica', 'Rachel', 'Ross']}
    `;
  }
```

And then say that you might want to include markup to you can do a bulleted list, so you can transform an array with `.map()`...

Reply:
Yes, that's a great suggestion! I did see that in the Lists doc section but didn't want to dedicate a whole step. Starting the example with that is a great way to bring it into the conversation.